sycl/test/warnings/sycl_2020_deprecations.cpp

```diff
@@ -148,7 +148,7 @@ int main() {
   sycl::ONEAPI::atomic_fence(sycl::ONEAPI::memory_order::relaxed,
                              sycl::ONEAPI::memory_scope::work_group);
 
-  // expected-warning@+1{{'INTEL' is deprecated: use 'ext::intel' instead}}
+  // expected-warning@+1{{'INTEL' is deprecated: use 'ext::intel::experimental' instead}}
```

Comment:
please add deprecation message check for the other deprecated interfaces.

Reply:
Done